content/posts/blog/xmpp-summit-21.md

```diff
@@ -0,0 +1,21 @@
+---
+title: XMPP Summit 21
+date: 2017-02-27 18:50
+author: Guus
+blog_id: blog
+---
+
+This year, the [XMPP Standards Foundation](https://xmpp.org) again gathered in force to attend the summit, that traditionally precedes the FOSDEM event in Brussels, Belgium. Barely fitting in the (rather sizable) room that was made available to us by Cisco, the XSF members had a fruitful two-day meeting. 
+
+The attending members, skillfully herded by Kev, addressed an impressive number of topics, including:
+
+* BIND2, improving the data exchange that occurs when an XMPP entity initially connects to a server.
+* MIX, the XEP that intends to be a replacement for the existing MUC protocol, bringing an up-to-date feature set and better extensibility.
+* Addressing the annoyance of instant-messaging-based spam.
+* The application of existing, or to be developed XMPP standards, to facilitate the growing world of Internet-of-Things.
```

Comment:
Might be sensible to point to https://wiki.xmpp.org/web/IoT_SIG so people can easily get to the right place.